sklearn/utils/_estimator_html_repr.py

```diff
@@ -466,15 +466,29 @@ class _HTMLDocumentationLinkMixin:
     Examples
     --------
     If the default values for `_doc_link_module`, `_doc_link_template` are not suitable,
-    then you can override them:
+    then you can override them and provide a method to generate the URL parameters:
     >>> from sklearn.base import BaseEstimator
-    >>> estimator = BaseEstimator()
-    >>> estimator._doc_link_template = "https://website.com/{single_param}.html"
+    >>> doc_link_template = "https://address.local/{single_param}.html"
     >>> def url_param_generator(estimator):
     ...     return {"single_param": estimator.__class__.__name__}
-    >>> estimator._doc_link_url_param_generator = url_param_generator
+    >>> class MyEstimator(BaseEstimator):
+    ...     _doc_link_module = "builtins"
```

Comment:
```suggestion
    ...     # use "builtins" since this is the module when declaring a class in
    ...     # this docstring
    ...     _doc_link_module = "builtins"
```